help/C/Help_ch_Transactions.xml

```diff
@@ -899,180 +905,543 @@ Author:
+          </listitem>
           <listitem>
-            <para><guilabel>Fixed width columns</guilabel>: Selecting this radio button will allow you to define 
-                 column boundaries by double clicking at the appropriate positions in the sample records displayed in the 
-                 panel below. Single clicking in a column will narrow, widen or merge the column.
+            <emphasis role="bold"><guilabel>Select File for Import</guilabel></emphasis>
+            <para>Display the File selection dialog. Navigate to the file you wish to import then click 
+              the <guibutton><emphasis role="bold">Next</emphasis></guibutton> button;
             </para>
           </listitem>
           <listitem>
-            <para><guilabel>Separators</guilabel>: Selecting this radio button will allow you to define characters
-                  which will be used to distinguish columns in the input file. The default is comma separated however spaces,
-                  tabs,colons or semicolons or any combination of them may be used to separate columns in the input file by 
-                  selecting the appropriate check boxes. You may also define custom separators by typing the required characters
-                  into the text box and selecting the <guilabel>Custom</guilabel> checkbox. This may be used in combination with any of the 
-                  predefined separators.
+            <emphasis role="bold">Import Preview</emphasis>
+            <para>The Import Preview pane illustrated below provides controls to set options for the import of CSV data and a preview of the data to be imported.
+              <screenshot id="CSV_Import_Preview">
+                <mediaobject>
+                  <imageobject role="html">
+                    <imagedata  fileref="figures/Help_CSV_Import_transactions_Preview.png" format="PNG" srccredit="David Cousens" width="515px"></imagedata>
+                  </imageobject>
+                  <imageobject role="fo">
+                    <imagedata  fileref="figures/Help_CSV_Import_transactions_Preview.png" format="PNG" srccredit="David Cousens" ></imagedata>
+                  </imageobject>
+                  <textobject>
+                    <phrase>Import Preview Pane</phrase>
+                  </textobject>
+                  <caption><emphasis role="bold">Import Preview Pane with settings to import the data shown.</emphasis></caption>
+                </mediaobject>
+              </screenshot> 
+              <itemizedlist>
+                <listitem>
+                  <emphasis role="bold"><guilabel>Load and Save Settings</guilabel></emphasis>
+                  <para id="trans-import-csv-save">Once you have set all the import parameters, you can save these settings by typing
+                    in a setting a unique name in the <guilabel>Load and Save Settings Entry</guilabel> combo box and clicking the
+                    <inlinemediaobject>
```

Comment:
You know the buttons depend on your desktops selected appearance aka "theme"?

Reply:
That makes life a bit more difficult.  OK will stick to a generic label description.  I guess somewhere we will need a disclaimer pointing out that gui element appearance in screenshots may  be OS and theme dependant and should not be relied upon.